inspec.yml

```diff
@@ -0,0 +1,10 @@
+name: apache-hardening
+title: Hardening Framework Apache Hardening Test Suite
+maintainer: Hardening Framework Team
+copyright: Hardening Framework Team
+copyright_email: hello@hardening.io
+license: Apache 2 license
+summary: Test-suite for best-practice apache hardening
+version: 0.9.0
```

Comment:
lets bump to version 2.0.0, since 1.0 was the serverspec test. I tagged it already.